gems/aws-sigv4/sig/signer.rbs

```diff
@@ -0,0 +1,50 @@
+module Aws
+  module Sigv4
+    class Signer
+      def initialize: (
+        service: String,
+        region: String,
+        access_key_id?: String,
```

Comment:
Same as https://github.com/aws/aws-sdk-ruby/pull/3152#discussion_r1870483168

`foo?: String` is not optional keyword syntax.
It support to call method like `m(foo?: 'bar')`. This is a very rare case.

Reply:
Sorry about that, I've updated the code.